### PR TITLE
Add Sunshine to Games - Administrative Utilities & Control Panels

### PR DESCRIPTION
**Pull request by arbdevml**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another project, please provide/link to a clear list of differences between both.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by kokomo123 — CHANGES_REQUESTED** [review]
I believe this can be included on the list as it's regularly maintained and the license is accurate, can you modify the description to say "with support up to" instead of "with support", thanks!

**Review by nodiscc — COMMENTED** [review]
> for Moonlight

What is Moonlight?

**Comment by kokomo123** [comment]
> > for Moonlight
> 
> What is Moonlight?

https://github.com/moonlight-stream/moonlight-docs/wiki/Frequently-Asked-Questions, per the FAQ it is a open source third party fork of what used to be the deprecated NVIDIA Gamestream, now SHIELD service, and Sunshine is a fork of that since NVIDIA Gamestream was deprecated but its still functional and their clients still are interoperable with eachother.

**Comment by nodiscc** [comment]
Thanks. I think if NVIDIA SHIELD streaming software is required to use this, the `⚠`  symbol should be added, as it is closed-source.

File: README.md [hunk 1]
```diff
@@ -1057,6 +1057,7 @@ Utilities for managing game servers.
 - [PufferPanel](https://www.pufferpanel.com/) - PufferPanel is an open source game server management panel, designed for both small networks and game server providers. ([Source Code](https://github.com/pufferpanel/pufferpanel)) `Apache-2.0` `Go`
 - [RconCli](https://github.com/gorcon/rcon-cli) - CLI for executing queries on a remote Valve Source dedicated server using the RCON Protocol. `MIT` `Go`
 - [SourceBans++](https://sbpp.github.io/) - Admin, ban, and communication management system for games running on the Source engine. ([Source Code](https://github.com/sbpp/sourcebans-pp)) `CC-BY-SA-4.0` `PHP`
+- [Sunshine](https://app.lizardbyte.dev/Sunshine/) - Remote game stream host for Moonlight with support 120 frames per second and 4K resolution. ([Source Code](https://github.com/LizardByte/Sunshine)) `GPL-3.0` `C++`
```
Comment (nodiscc) [inline README.md:1060]:
```suggestion
- [Sunshine](https://app.lizardbyte.dev/Sunshine/) `⚠` - Remote game stream host for Moonlight with support 120 frames per second and 4K resolution. ([Source Code](https://github.com/LizardByte/Sunshine)) `GPL-3.0` `C++`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by kokomo123** [comment]
> Thanks. I think if NVIDIA SHIELD streaming software is required to use this, the `⚠` symbol should be added, as it is closed-source.

Its not required as sunshine is FOSS, moonlight client im not sure about.

**Comment by nodiscc** [comment]
It still requires NVIDIA hardware/SHIELD/GeForce Experience, which is closed-source (unless I misunderstand). So the  `⚠` symbol applies.

**Comment by kokomo123** [comment]
You would be right then, it does require a nvidia GPU (which the closed source drivers are still the stable ones before the nvidia open source ones get mature enough) and it has AMD support, anyway, i agree with this change.

**Comment by arbdevml** [comment]
> I believe this can be included on the list as it's regularly maintained and the license is accurate, can you modify the description to say "with support up to" instead of "with support", thanks!

Thank you for your review, the description was modified.

**Comment by arbdevml** [comment]
> Thanks. I think if NVIDIA SHIELD streaming software is required to use this, the `⚠` symbol should be added, as it is closed-source.

Moonlight - open-source game streaming client for different platforms.
Sunshine is an open-source implementation of Nvidia proprietary stream software.
Sunshine replaces proprietary streaming host software.


For example: 
windows rdp client (mstsc) - foss alternative remmina
windows rdp sever - foss alternative xrdp
remmina and xrdp are both foss.

Moonlight and Sunshine are free libre open-source software.

**Comment by nodiscc** [comment]
@arbdevml thanks for clarifying

**Review by nodiscc — APPROVED** [review]

**Review by kokomo123 — APPROVED** [review]
Thanks again for clarifying as well, that would be a better explanation then mine.

**Pull request merged**